gui/src/components/onboarding/pages/body-proportions/scaled-steps/ManualHeightStep.tsx

```diff
@@ -65,13 +66,14 @@ export function ManualHeightStep({
   );
 
   const submitHmdHeight = (values: HeightForm) => {
-    setHmdHeight(values.height);
+    const newHeight = values.height * 0.936;
```

Comment:
I kinda want to know where this magic multiplier came from? Either way, it should be a const value somewhere and properly named. Do not magic numbers, please!

Reply:
Pulled it to a constant and added a comment (plus updated the existing one on the server side a little)